src/seq_mv/csr_matvec.c

```diff
@@ -1126,3 +1126,186 @@ hypre_CSRMatrixMatvec_FF( HYPRE_Complex    alpha,
 
    return ierr;
 }
+
+/*--------------------------------------------------------------------------
+ * hypre_CSRMatrixInvDiagAxpy
+ *
+ * y = alpha*inv(A_D)*x + beta*y
+ *--------------------------------------------------------------------------*/
+
```

Comment:
Combine with `hypreDevice_DiagScaleVector` that is the device version

Reply:
I was talking to @rfalgout and we will phase out the `InvDiag` functions that I added here and use the `DiagScaleVector` function instead to reduce code duplication. I think we still need some discussion on the name of the function that performs "element division" for vectors, right @rfalgout?